Tests/MMELocationManagerTests.mm

```diff
@@ -420,11 +420,6 @@ - (void)stopMonitoringRegions;
                     [locationManager startUpdatingLocation];
                     [locationManager locationManager:locationManagerInstance didUpdateLocations:@[movingLocation]];
                     locationManager.backgroundLocationServiceTimeoutTimer should_not be_nil;
-                });
-                
-                it(@"tells the location manager to start monitoring for region", ^{
-                    expectedRegion.notifyOnEntry = NO;
-                    expectedRegion.notifyOnExit = YES;
```

Comment:
Why are we removing these configurations from the expected region? Is the region being returned different?

Reply:
Because they are set in `establishRegionMonitoringForLocation` which is called by `locationManager:didUpdateLocations` in the beforeEach block.